straxen/plugins/daqreader.py

```diff
@@ -22,45 +24,63 @@ class ArtificialDeadtimeInserted(UserWarning):
 
 @export
 @strax.takes_config(
+
+    # All these must have track=False, so the raw_records hash never changes!
+
+    # DAQ settings -- should match settings given to redax
+    strax.Option('record_length', default=110, track=False, type=int,
+                 help="Number of samples per raw_record"),
+    strax.Option('digitizer_sampling_resolution',
+                 default=10, track=False, type=int,
+                 help="Digitizer sampling resolution"),
+    strax.Option('run_start_time', type=float, track=False, default=0,
+                 help="time of start run (s since unix epoch)"),
     strax.Option('daq_chunk_duration', track=False,
                  default=int(5e9), type=int,
                  help="Duration of regular chunks in ns"),
     strax.Option('daq_overlap_chunk_duration', track=False,
                  default=int(5e8), type=int,
                  help="Duration of intermediate/overlap chunks in ns"),
-    strax.Option('daq_input_dir', type=str, track=False,
-                 help="Directory where readers put data"),
+    strax.Option('daq_compressor', default="blosc", track=False,
```

Comment:
Is there a specific reason we're still specifying blosc as the default compressor for live data? Redax uses lz4 by default, and has for several months at this point.

Reply:
Thanks, should be fixed now